packages/api/src/queueAdapters/base.ts

```diff
@@ -24,12 +25,17 @@ export abstract class BaseAdapter {
     this.allowCompletedRetries = this.allowRetries && options.allowCompletedRetries !== false;
     this.prefix = options.prefix || '';
     this.description = options.description || '';
+    this.category = options.category || 'default';
   }
 
   public getDescription(): string {
     return this.description;
   }
 
+  public getCategory(): string {
+    return (this.category || 'default').toUpperCase();
```

Comment:
Why the uppercase?

Reply:
The ideia is to normalize the data, otherwise `Category 1` and `category 1` will be distinct.

Reply:
I am going to remove uppercase here and treat it in another leve, since the `category` property at base adapter is public, otherwise `getCategory` and `class.category` can return different data.